packages/common/src/server-connection.js

```diff
@@ -54,7 +54,9 @@ export const Provider = ({ Pusher, pusherKey, debug = false, children }) => {
       },
     });
 
-    const channel = pusher.subscribe(`private-${user.id}`);
+    const nonce = crypto.getRandomValues(new Uint32Array(1))[0];
```

Comment:
reused the same code we use to generate the nonce on sign-in, lmk if ok

Reply:
Yas!